crates/holochain_conductor_api/src/admin_interface.rs

```diff
@@ -38,6 +38,13 @@ pub enum AdminRequest {
     /// [`AdminResponse::DnaRegistered`]
     RegisterDna(Box<RegisterDnaPayload>),
 
+    /// Request the definition of a DNA.
+    ///
+    /// # Returns
+    ///
+    /// [`AdminResponse::DnaDefRequested`]
+    RequestDnaDef(Box<DnaHash>),
```

Comment:
Just a naming thing, I would call this `GetDnaDef`. A request to make a request is like asking if I can ask you a question :)